services/src/main/java/org/fao/geonet/api/pages/PagesAPI.java

```diff
@@ -407,6 +424,25 @@ public ResponseEntity<List<org.fao.geonet.api.pages.PageProperties>> listPages(
                         filteredResult.add(new org.fao.geonet.api.pages.PageProperties(page));
                     }
                 }
+            } else if (page.getStatus().equals(Page.PageStatus.PRIVATE_GROUP) && us.getProfile() != null && us.getProfile() != Profile.Guest) {
+                String myUserId = us.getUserId();
```

Comment:
Suggest to rename variable to `currentUserId` as it is clearer.

Reply:
renamed